.circleci/config.yml

```diff
@@ -31,6 +48,18 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+          requires:
+            - "template-chart"
+      
+      - architect/run-tests-with-ats:
+          name: run-chart-tests-with-ats
+          filters:
+            # Do not trigger the job on merge to master.
+            branches:
+              ignore:
+                - master
```

Comment:
You mean main right?

Reply:
I would have preferred but unfortunately, this repo's "main" branch is called "master" :/

Reply:
Is it? Maybe we should change it then :D

Reply:
I would totally agree :)